app/views/immunisation_imports/index.html.erb

```diff
@@ -63,9 +63,11 @@
 </div>
 
 <% content_for :after_main do %>
-  <%= render(AppDevToolsComponent.new) do %>
-    <%= govuk_button_to "Download DPS export", dps_export_campaign_reports_path(@campaign) %>
+  <% if Flipper.enabled? :dev_tools %>
```

Comment:
Could the feature flag check be part of the `AppDevToolsComponent` component? Would we ever want the component to render without the dev tools flag being enabled?

We [already have a `render?` method on that component that checks that we're not running in production](https://github.com/nhsuk/manage-vaccinations-in-schools/blob/940f20284bd622ad5382ab58884cc7202c4e6601/app/components/app_dev_tools_component.rb#L18-L20).

Reply:
@thomasleese like this? [828f0bb](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/1590/commits/828f0bbf985bf1ff644c77b8015f1cfb1944a520)